src/main/kotlin/com/ondosee/global/config/FcmConfig.kt

```diff
@@ -0,0 +1,40 @@
+package com.ondosee.global.config
+
+import com.google.auth.oauth2.GoogleCredentials
+import com.google.firebase.FirebaseApp
+import com.google.firebase.FirebaseOptions
+import com.ondosee.global.config.properties.FcmProperties
+import org.springframework.context.annotation.Configuration
+import java.io.File
+import java.net.URL
+import java.nio.file.Files
+import java.nio.file.Paths
+import javax.annotation.PostConstruct
+
+@Configuration
+class FcmConfig(
+    private val fcmProperties: FcmProperties
+) {
+    companion object {
+        const val PATH = "./credentials.json"
+    }
+
+    @PostConstruct
+    fun init() {
+        runCatching {
+            URL(fcmProperties.url).openStream().use {
+                Files.copy(it, Paths.get(PATH))
+                val file = File(PATH)
+                if (FirebaseApp.getApps().isEmpty()) {
+                    val options = FirebaseOptions.builder()
+                        .setCredentials(GoogleCredentials.fromStream(file.inputStream()))
+                        .build()
+                    FirebaseApp.initializeApp(options)
+                }
```

Comment:
```suggestion
                if (FirebaseApp.getApps().isEmpty()) {
                   FirebaseOptions.builder()
                        .setCredentials(GoogleCredentials.fromStream(file.inputStream()))
                        .build()
                        .run(FirebaseApp::initializeApp)
                }
```

이렇게 run을 사용해서 변수를 저장하지 않고 처리해버리는 건 어떤가요?

Reply:
[c3df07f](https://github.com/ondosee/ondosee-Backend/pull/15/commits/c3df07ff55d6c98abdfca9bdabedb07d4bf82cfd)

수정했습니다!!